### PR TITLE
Ucp cluster metrics

### PR DESCRIPTION
**Pull request by bermudezmt**
### Proposed changes

- Added note around SELinux Prometheus deployment on a worker node.

### Related issues (optional)

Fixes https://github.com/docker/docs-private/issues/910

### CODE REVIEW

File: ee/ucp/admin/configure/collect-cluster-metrics.md [hunk 1]
```diff
@@ -84,8 +84,12 @@ To deploy Prometheus on worker nodes in a cluster:
     $ kubectl label node 3a724a-1 ucp-metrics=
     node "test-3a724a-1" labeled
     ```
+    
+     > **SELinux Prometheus Deployment**
```
Comment (bermudezmt) [inline ee/ucp/admin/configure/collect-cluster-metrics.md]: @wsong should we add UCP versions this note  is relevant to given [this comment](https://github.com/docker/orca/issues/15590#issuecomment-446024780)?

Comment (wsong) [reply ee/ucp/admin/configure/collect-cluster-metrics.md]: Sure, that's a good idea.

File: ee/ucp/admin/configure/collect-cluster-metrics.md [hunk 2]
```diff
@@ -84,8 +84,12 @@ To deploy Prometheus on worker nodes in a cluster:
     $ kubectl label node 3a724a-1 ucp-metrics=
     node "test-3a724a-1" labeled
     ```
+    
+     > **SELinux Prometheus Deployment for UCP 3.1.0-3.1.2**
```
Comment (bermudezmt) [inline ee/ucp/admin/configure/collect-cluster-metrics.md]: @L-Hudson I'm torn between leaving the versioning like this or specifically call out `UCP 3.1.0, 3.1.1, and 3.1.2` given how this note will not apply to `UCP 3.1.3`.

Comment (ahh-docker) [reply ee/ucp/admin/configure/collect-cluster-metrics.md]: Fixed

**Pull request merged**